GVFS/GVFS.FunctionalTests/Tests/GitCommands/StatusTests.cs

```diff
@@ -136,7 +137,13 @@ public void ModifyingHeadRefInvalidatesCache()
         [TestCase]
         public void AppendFile()
         {
-            this.AppendAllText("More Data", @"Readme.md");
+            BashRunner bash = new BashRunner();
+            string filePath = @"Readme.md";
+            string content = "Apended Data";
+            string virtualFile = Path.Combine(this.Enlistment.RepoRoot, filePath);
+            string controlFile = Path.Combine(this.ControlGitRepo.RootPath, filePath);
+            bash.AppendAllText(virtualFile, content);
+            bash.AppendAllText(controlFile, content);

```

Comment:
Do we have a test that validates that the contents of the file after using `>>` is correct?